cmd/podman/parse.go

```diff
@@ -816,3 +816,12 @@ func getLoggingPath(opts []string) string {
 	}
 	return ""
 }
+
+// validateFileName returns an error if filename container ":"
+// as it is currently not supported
+func validateFileName(filename string) error {
+	if strings.Contains(filename, ":") {
+		return errors.Errorf("invalid filename, should not contain ':' %q", filename)
```

Comment:
Can you put the "should not contain ':'..." part in parentheses, to separate it from the filename we're validating?

Reply:
Fixed